src/winetricks

```diff
@@ -718,6 +718,15 @@ w_expand_env()
     winetricks_early_wine cmd.exe /c echo "%$1%"
 }
 
+# Check if $1 is executable (output custom message using w_debug if $2 is specified)
+w_check_exec()
+{
+    if ! command -v "$1" >/dev/null; then
+        [ -z "$2" ] && { printf "Command '%s' is not executable\n" "$1" && exit 1 ;} || printf '%s\n' "$2"
+    elif command -v "$1" >/dev/null; then w_debug "Command '$1' is executable"
```

Comment:
Please keep the style consistent (i.e., move the statement to its own, indented line).

Reply:
Resolved in https://github.com/Winetricks/winetricks/pull/1351/commits/e9cb0482ef5067adee136edc17e5bd2de73d2d4b